src/integrationTest/groovy/wooga/gradle/unity/UnityIntegrationSpec.groovy

```diff
@@ -139,12 +139,14 @@ abstract class UnityIntegrationSpec extends IntegrationSpec {
                 @echo off
                 echo ${mockUnityStartupMessage}
                 echo %*
+                echo env
```

Comment:
is `echo env` the same on windows?

Reply:
No it's not;; I will look for the right invocation.

Reply:
It's `set` as far as I can remember.

Reply:
Aye that's correct.